x-pack/plugins/observability/public/pages/alerts/alerts_table_t_grid.tsx

```diff
@@ -382,11 +372,12 @@ export function AlertsTableTGrid(props: AlertsTableTGridProps) {
     }
   }, []);
 
-  const leadingControlColumns = useMemo(() => {
+  const leadingControlColumns: ControlColumnProps[] = useMemo(() => {
     return [
       {
         id: 'expand',
         width: 120,
+        showCheckboxes: false,
```

Comment:
@fkanout We don't need this code. The fix is done by passing `showCheckboxes` in the tGridProps https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/pages/alerts/alerts_table_t_grid.tsx#L421 as you do as part of this ticket https://github.com/elastic/kibana/pull/119608

Reply:
Correct, done.